### ISSUE TITLE
[BUG/ISSUE] jupyterlab installation causes GCPy to break

### ISSUE THREAD

**Issue by hmhorow** [geoschem/gcpy#201]
Hannah Horowitz from UIUC here!
## Describe the bug
I installed gcpy following the read the docs instructions. I did this 2x. in one conda environment, I did not install jupyterlab and plots showed up just fine using the command line. In the other conda environment, I installed jupyterlab. then, when trying to plot anything, there were numerous ipython and matplotlib errors. 

## To Reproduce
List the steps to reproduce the bug or issue below:

conda env create -n gcpy_new --file=gcpy_environment.yml
conda deactivate
conda activate gcpy_new
conda install geoschem-gcpy
conda deactivate
conda activate gcpy_new
conda install jupyterlab

```python
import xarray as xr
import gcpy.plot as gcplot
import matplotlib.pyplot as plt
ds = xr.open_dataset('/projects/horowitz_group/hmhorow/hannah/GEOSChem/rundirs/gc_4x5_fullchem/OutputDir/HEMCO_diagnostics.201907010000.nc')
gcplot.single_panel(ds['EmisISOP_Biogenic'], title='Biogenic isoprene emissions over the US', extent=[-130, -60, 20, 50])
plt.show()
```

## Expected behavior
Figure would plot. does in the environment without jupyter. Causes many errors in the one with jupyterlab installed. 

## Required information:
```python
>>> import gcpy
>>> gcpy.show_versions()
```
```
AttributeError: module 'gcpy' has no attribute 'show_versions'
```
This doesn't work in either environment, the one that plots figures just fine, or the one with numerous errors. There are many other attributes, just not this one. 

## Additional context
Full error message here: https://docs.google.com/document/d/1grJ4QXEDd70-uMqUBiV4rMzAaTI0TRkk79Uiu_6XyzE/edit?usp=sharing 
It seems that earlier versions of gcpy perhaps contained Jupyter and/or IPython (see #36  and #59  ) but it appears to no longer be the case.

**Comment by yantosca** [comment]
Hi @hmhorow, thanks for writing.  @lizziel is currently updating the GCPy docs with updated installation instructions, as we have realized that the current instructions are out of date.

Also -- one thing we have discovered is that sometimes when using Conda to install a package, it will automatically try to update dependent packages to different versions.  This can cause issues.  In the `environment.yml` file that comes with GCPy, we have started to request specific package versions and to use Python 3.9.

You might also try installing jupyterlab via PyPi: `pypi install jupyterlab` into the `gcpy_new` environment.  That might work better.  We don't typically use jupyter ourselves so we haven't tested.  If you find a version that works then let us know and we can add that specific version to `environment.yml`.

**Comment by yantosca** [comment]
Also see the latest docs: https://gcpy.readthedocs.io/en/latest/

**Comment by yantosca** [comment]
Hi @hmhorow.  Just getting back to this issue.  In [GCPy 1.3.3](https://github.com/geoschem/gcpy/releases/tag/1.3.3), we have updated the installation instructions to state that Conda 4.12.0 or earlier is needed to install GCPy.  As @lizziel described in #202, updating to a newer Conda environment leads to package compatibility issues.

Also I restored `ipython` and `jupyter` to the GCPy `environment.yml` file that ships with version 1.3.3.  That will make it easier to use Jupyter notebooks from within the GCPy python environment.
